apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/config/SerialConsistencyValidatorTest.java

```diff
@@ -0,0 +1,63 @@
+package io.stargate.sgv2.api.common.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.stargate.bridge.proto.QueryOuterClass;
+import java.util.Set;
+import java.util.function.Predicate;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import javax.validation.constraints.NotNull;
+import org.junit.jupiter.api.Test;
+
+public class SerialConsistencyValidatorTest {
+
+  private record TestRecord(
+      @NotNull
```

Comment:
remove not null and add test with `null`, must be valid

Reply:
@NotNull removed and null is handled in the validator.